saas/views/billing.py

```diff
@@ -211,12 +211,16 @@ def get(self, request, *args, **kwargs):
         else:
             auth_code = request.GET.get('code', None)
             if auth_code:
-                self.object = self.get_object()
-                self.object.processor_backend.connect_auth(
-                    self.object, auth_code)
-                self.object.save()
-                messages.success(self.request, _("Connection to your deposit"\
-                    " account was successfully updated"))
+                try:
+                    self.object = self.get_object()
+                    self.object.processor_backend.connect_auth(
+                        self.object, auth_code)
+                    self.object.save()
+                    messages.success(self.request, _("Connection to your deposit"\
+                        " account was successfully updated"))
+                except:
```

Comment:
Please catch only `ProcessorError` here. Also add a `LOGGER.exception` statement so we are notified something went wrong. Thank you.